### ISSUE TITLE
Fix estimate figures in the Clearinghouse (via API)

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#190]
>>     Some estimates figures are wrong in monthly edition. They did not match figures in AC19. But, we did not update any of such figures since ac19. @gdower What we can do about this? (If nothing, probably we should kill all estimates in CoL).

Example:
ac19: Phylum Tracheophyta • 345,676 of 348,000 est. living spp
now: Phylum Tracheophyta • 348,689 of 340,000 est. living spp
(Michael Hassler pointed this with critics. The problem is: I have no idea how many estimates incorrect and does not match ac19).

**Comment by yroskov** [comment]
@gdower wrote a query to look for estimates that differ between now and ac19. **109 records are different.**

YR: My guess, Ruud replaced latest version of estimates DB with version from his archive. It was happened after ac19, during migration at Naturalis. (Different version of user interface was also applied to ac19 and some other earlier editions. This version lost column "Kingdom" in a list of GSDs at Source Databases page).

**Comment by yroskov** [comment]
How to replace 109 figures and related sources in the Clearinghouse with figures from ac19?

**Comment by gdower** [comment]
I'm writing a script to fix the estimates right now.

**Comment by mdoering** [comment]
if useful we could also query the db to compare estimates with actual counts and report those that do not match up.
Or just report all of them side by side - that could even become a regular API / UI tool

**Comment by yroskov** [comment]
We need an interface option to populate actual counts in estimates. Only few GSDs agreed that their actual species counts are equal to estimates.

**Comment by yroskov** [comment]
@mdoering **Does CoLDP format allows us to update estimate figures during sync, if GSD provide them?**

@gdower & @yroskov would like to update Hassler's figures in a new release. Hassler's spreadsheets contain estimates for orders (WF), families (WF&WP) and genera (WF&WP), and Geoff crawling these figures in CoLDP.

**Comment by mdoering** [comment]
yes, [ColDP allows that](https://github.com/CatalogueOfLife/coldp/blob/master/README.md#speciesestimate). But we have not implemented the importing of such estimates yet. Not a big thing but so far not on our priority list

**Comment by mdoering** [comment]
see https://github.com/CatalogueOfLife/backend/issues/838 for open coldp importer issues
